test/unit/puppetlabs/services/request_handler/request_handler_core_test.clj

```diff
@@ -121,7 +121,8 @@
   (testing "request with binary content type does not consume body"
     (let [body-string "some random text"]
       (doseq [content-type [nil "" "application/octet-stream"
-                            "APPLICATION/octet-Stream"]]
+                            "APPLICATION/octet-Stream"
```

Comment:
I couldn't find a higher level test after a quick search. I think I'm personally ok if this just stays a unit test. It runs it through the `core/wrap-params-for-jruby` fn, and all requests that use a jruby instance are wrapped by that fn  [here](https://github.com/puppetlabs/puppetserver/blob/master/src/clj/puppetlabs/services/request_handler/request_handler_core.clj#L268)